pyte/screens.py

```diff
@@ -186,40 +186,27 @@ def __repr__(self):
     def display(self):
         """A :func:`list` of screen lines as unicode strings."""
         def render(line):
-            it = iter(line)
-            while True:
-                char = next(it).data
+            wide_char = False
```

Comment:
True, thanks for the explanation!

One minor thing: I'd prefer `is_` prefix for boolean variables.